### PR TITLE
Adds extra labcoats to medical wardrobe

### PR DESCRIPTION
**Pull request by ghost**
:cl: Mark9013100
add: Medical Wardrobes now contain an additional standard and EMT labcoat.
/:cl:

[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)

For consistency, there's three jumpsuits, pairs of shoes, and even 3 EMT caps, but only two of the labcoats.

Also fixes spaced (src)s.

### CODE REVIEW

File: code/game/objects/structures/crates_lockers/closets/job_closets.dm [hunk 1]
```diff
@@ -215,17 +215,17 @@
 	new /obj/item/storage/backpack/medic(src)
 	new /obj/item/storage/backpack/satchel/med(src)
 	new /obj/item/clothing/suit/hooded/wintercoat/medical(src)
-	new /obj/item/clothing/under/rank/nursesuit (src)
+	new /obj/item/clothing/under/rank/nursesuit(src)
 	new /obj/item/clothing/head/nursehat (src)
```
Comment (ShizCalev) [inline code/game/objects/structures/crates_lockers/closets/job_closets.dm]: space here too.

**Review by ShizCalev — DISMISSED** [review]

File: code/game/objects/structures/crates_lockers/closets/job_closets.dm (hunk 1 above)
Comment (Mark9013100) [reply code/game/objects/structures/crates_lockers/closets/job_closets.dm]: I'm half blind, will fix.

**Comment by Bawhoppen** [comment]
Resource creep

This is why nobody ever orders anything like clothes in cargo

**Comment by Mark9013100** [comment]
I wasn't aware you could order departmental clothing from cargo, aside from security.

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**